packages/solvuu_build/solvuu_build.0.0.1/descr

```diff
@@ -0,0 +1 @@
+Solvuu's build system.
```

Comment:
How about I update the README, which is more likely to get attention and be maintained.